dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentCodeGenerator.java

```diff
@@ -0,0 +1,97 @@
+package com.dotcms.experiments.business;
+
+import static com.dotmarketing.util.FileUtil.getFileContentFromResourceContext;
+
+import com.dotmarketing.beans.Host;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.Logger;
+import java.io.IOException;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * Util class to generate the JS/HTML code needed to Support Experiment into the Browser.
+ */
+public enum ExperimentCodeGenerator {
+    INSTANCE;
+
+    /**
+     * Return the Experiment Js Code to inject when there are not any Experiment Running
+     * @return
+     */
+    private String getNotExperimentRunningJSCode() {
+        return "<SCRIPT>localStorage.removeItem('experiment_data');</SCRIPT>";
```

Comment:
This value is widely used, not only in this PR but also here: https://github.com/dotCMS/core/pull/24357/files#diff-aac9fd429a1e8f9ab8efad97d4259f8be79b41078a9c0750704c907760f2d85cL2258. It would be good to centralize it in a public constant.

Reply:
Yes, the test should reference the production code.
In the case it changes we don't need to change it on every class/test.